data/sql_updates/committee_history.sql

```diff
@@ -32,53 +32,54 @@ select distinct on (fec_yr.cmte_id, fec_yr.fec_election_yr)
     fec_yr.cmte_nm as name,
     fec_yr.tres_nm as treasurer_name,
     to_tsvector(fec_yr.tres_nm) as treasurer_text,
-    dcp.org_tp as organization_type,
-    expand_organization_type(dcp.org_tp) as organization_type_full,
-    dcp.expire_date as expire_date,
+    f1.org_tp as organization_type,
+    expand_organization_type(f1.org_tp) as organization_type_full,
+    --verify, but this column is gone now
+    null as expire_date,
```

Comment:
No use of it on the front-end.  We can ask Paul if that's ok; if so then maybe drop a line in the release notes that it is being deprecated.